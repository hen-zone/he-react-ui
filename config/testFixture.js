import createTestContext from 'react-cosmos-test/enzyme';

export default function testFixture(fixture, variant) {
  const { mount, getWrapper } = createTestContext({ fixture });
  const { component } = fixture;
  const { displayName, name } = component;
  mount();
  test(`<${displayName ||
    name} /> rendered correctly with ${variant} fixture`, () =>
    expect(getWrapper()).toMatchSnapshot());
}
