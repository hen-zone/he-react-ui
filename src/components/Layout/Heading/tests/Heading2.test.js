import fixture from '../fixtures/Heading2.fixture.js'
import testFixture from '../../../../../config/testFixture'

testFixture(fixture, 'Heading2')
