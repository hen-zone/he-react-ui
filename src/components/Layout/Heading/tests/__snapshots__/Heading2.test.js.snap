// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> rendered correctly with Heading2 fixture 1`] = `
<Heading
  h1={false}
  h2={true}
  h3={false}
  h4={false}
  nav={false}
>
  <h2
    className="heading h2"
  >
    Heading
  </h2>
</Heading>
`;
